content/docs/latest/installing/vms/hyper-v.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Running Flatcar Container Linux on Hyper-V
+linktitle: Running on Hyper-V
+weight: 30
+---
+
+_While we always welcome community contributions and fixes, please note that Hyper-V is not an officially supported platform at this time because the release tests don't run for it. (See the [platform overview](/#installing-flatcar).)_
+
+These instructions will walk you through running Flatcar Container Linux on Hyper-V.
+
+## Choose a channel
+
+Flatcar Container Linux is designed to be updated automatically with different schedules per channel. You can [disable this feature][update-strategies], although we don't recommend it. Read the [release notes][release-notes] for specific features and bug fixes.
+
+Hyper-V images are created only for AMD64 and come in two disk formats, VHD for Hyper-V VM Generation 1 and VHDX for both Hyper-V Generation 1 and 2 VMs.
+
+Note: Hyper-V Generation 2 virtual machines need to have secure boot disabled, as the VHDX images are not signed.
+
+How to download a VHDX image file:
+
+```powershell
+# VHDX image is available for download from the current Alpha onwards
+$imageUrl = "https://alpha.release.flatcar-linux.net/amd64-usr/current/flatcar_production_hyperv_vhdx_image.vhdx.bz2"
+$wc = New-Object System.Net.WebClient
+$wc.DownloadFile($url, $imageUrl.split("/"[-1]))
+# 7zip or another decompression tool that supports .bz2 is required
+ & 'C:\Program Files\7-Zip\7z.exe' x flatcar_production_hyperv_vhdx_image_v2.vhdx.bz2
+```
+## Creating a config-drive
```

Comment:
we should have ignition as an example first as the preferred approach and config-drive later